test/integration/rollbar_int_test.go

```diff
@@ -0,0 +1,17 @@
+package integration
+
+import (
+	"testing"
+
+	"github.com/ActiveState/cli/internal/constants"
+	"github.com/ActiveState/cli/internal/testhelpers/e2e"
+)
+
+func TestRollbarReportConfig(t *testing.T) {
+	ts := e2e.New(t, true)
+	defer ts.Close()
+
+	cp := ts.Spawn("config", "set", constants.ReportErrorsConfig, "false")
+	cp.Expect("Successfully")
+	cp.ExpectExitCode(0)
+}
```

Comment:
I think having a test for this is only really worth it if we also test that the config change had an effect. Otherise you're basically just testing whether `config set` works.

I'd consider this out of scope for the story, so feel free to disregard the test instead. Don't think we have any existing code that tests rollbar events.

Reply:
Well considering this ticket is all about the fact that `config set` didn't work for `report.errors`, I thought this was a worthwhile test. If a regression happens, it'll get caught. I agree that ideally it would test functionality, but I don't see an easy way to do that.

Reply:
Per our zoom call, I have removed this.